### DISCUSSION TITLE
Reuse previous studies after adding new parameters

### DISCUSSION THREAD

**Discussion by charleskhazoom** [optuna/optuna#5666] [Q&A]
In Optuna, let's say I  completed a study that optimized over 2 parameters A, B. There is a third parameter C that was not yet exposed to Optuna but was equal to 1 internally for instance. Now I want to add C and optimize over A, B, C. Is there a way to use my previous study's database which only optimized over A, B ?

**Comment by nzw0301** [comment]
In general, it depends on how A and B affect the new objective using A, B, and C. If the previous one and the new one are quite similar, which means C is not important for the new objective, we can use A and B values from the previous study like https://optuna.readthedocs.io/en/stable/reference/generated/optuna.study.Study.html#optuna.study.Study.enqueue_trial.
